docs/storage-and-interfaces.md

```diff
@@ -4,13 +4,8 @@ To be updated and specialized for the ejprd use case.
 # Set up starter-kit-storage-and-interfaces in demo mode
 
 1. Download the repo https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces/
-1. Go into the repo's root folder `cd starter-kit-storage-and-interfaces`
-1. Run the following commands, to sure that all configuration files are in place.
-    ```shell
-    cp ./config/config.yaml.example ./config/config.yaml
-    cp ./config/iss.json.example ./config/iss.json
-    cp ./.env.example ./.env
-    ```
+1. Go into the repo's root folder `cd starter-kit-storage-and-interfaces`.
+1. Check out branch `feat/htsget-crypt4gh`.
```

Comment:
This branch changes the docker-compose.yml to rely on a docker image registry that I don‘t have access to. Is there a public version on these images?

Reply:
@wna-se does that refer to the htsget version? I will change access to public for pulling images

Reply:
Malin already fixed this somehow, I was able to get the images after pulling down her updates.

Reply:
Ah very nice. Just a heads up that there are some changes implemented today in htsget that are not reflected in the image we have in the registry currently. We will update as soon as possible